doc/quickstart.rst

```diff
@@ -0,0 +1,96 @@
+Quickstart
+=====================
+
+What is xgcm for?
+-----------------
+.. raw:: html
+
+    <div style="position: relative; padding-bottom: 10.25%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/lo6JvrJ-YA8" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    </div>
+
+What ingredients do I need to start with xgcm?
+----------------------------------------------
+1. An :ref:`installed <installation>` version of xgcm
+2. Your xarray dataset of choice `ds`.
+3. Information on how the grid of `ds` is layed out. This is often encoded in metadata or you need to infer it from the dataset (see more below).
+4. (Optional) All 'metrics' you have available in the dataset (think: distances, areas, volumes). You need these if you want to use the more awesome features.
+
+
+How to get started - Step by step
+---------------------------------
+The `Grid` object is essential to the way xgcm works. It stores information about the grid setup for you. 
+`Grid` knows about the logical [axes], which dimensions and metrics (represented by xarray [coordinates]) belong to each axis.
```

Comment:
Why `[coordinates]` in brackets. Is this supposed to be a link?

Reply:
I think I wanted to link to xarray, but maybe that is a bit too much?

Reply:
I put the link in. Cant have too many of those.